redfish-core/lib/account_service.hpp

```diff
@@ -105,6 +106,7 @@ inline std::string getRoleIdFromPrivilege(std::string_view role)
     }
     return "";
 }
+
```

Comment:
be careful adding empty lines, makes reviewing harder

Reply:
ok, I like to add space between two function